internal/restorer/fileswriter.go

```diff
@@ -127,12 +128,10 @@ func (w *filesWriter) writeToFile(path string, blob []byte, offset int64, create
 
 // openFileWithCreate opens the file with os.O_CREATE flag along with os.O_TRUNC and os.O_WRONLY.
 func openFileWithCreate(path string) (file *os.File, err error) {
```

Comment:
Please also fully inline `openFileWithCreate` and `openFileWithTruncWrite` into the calling functions. To understand `writeToFile` it is necessary to know the exact implementation of both functions, so it's easier to understand if the `os.OpenFile` calls are directly included in `writeToFile` and are not part of a helper function.

Reply:
Done